test/intl402/Collator/prototype/toStringTag/toString-removed-tag.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2020 Alexey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-intl.collator.prototype-@@tostringtag
+description: >
+  Object.prototype.toString doesn't special-case neither Intl.Collator instances nor its prototype.
+info: |
+  Object.prototype.toString ( )
+
+  [...]
+  14. Else, let builtinTag be "Object".
+  15. Let tag be ? Get(O, @@toStringTag).
+  16. If Type(tag) is not String, set tag to builtinTag.
+  17. Return the string-concatenation of "[object ", tag, and "]".
```

Comment:
I would be nice to add a test that overrides the `@@toStringTag` of these constructors.

PTAL @ [`symbol-tag-override-bigint.js`](https://github.com/tc39/test262/blob/master/test/built-ins/Object/prototype/toString/symbol-tag-override-bigint.js) in the `test/built-ins/Object/prototype/toString/` folder.

Reply:
Added tests in 74a7951.

`Intl.NumberFormat` is already covered by [`test/intl402/NumberFormat/prototype/toStringTag/configurable.js`](https://github.com/tc39/test262/blob/master/test/intl402/NumberFormat/prototype/toStringTag/configurable.js).